### DISCUSSION TITLE
TLS options not applied

### DISCUSSION THREAD

**Discussion by daMatz** [mitmproxy/mitmproxy#5545] [Q&A] [answered]
Unfortunately the TLS options seem to be not applied, when setting them in the mitmproxy options.
Can you tell me, what I'm doing wrong here?

At frst I only tried to adapt the `ciphers_server` option, but that didn't work.

Then I read [here](https://github.com/mitmproxy/mitmproxy/discussions/5539), that setting them for TLS1.3 is not supported yet, therefore my plan was to force TLS1.2 by applying the following options in my `config.yaml`:

```
ciphers_server: ECDHE-ECDSA-AES128-GCM-SHA256
tls_version_server_max: TLS1_2
```

However, when sniffing the `Client Hello` to `https://example.com` in Wireshark, I get the following list of supported ciphers:

![capture2](https://user-images.githubusercontent.com/9104870/185928008-273fd86c-c25a-42e2-86c3-37dffa006da2.PNG)

I also tried the ciphers `AES128-SHA` and `TLS_AES_256_GCM_SHA384` to make sure there is no formatting issue when setting them with OpenSSL.

**Comment by mhils** [comment, marked as answer]
Fixed in #5546!

**Reply by daMatz** [reply]
Nice, thank you 👍 
Edit: Just confirmed with current dev version: it is now working as expected.
